docs/syntax-reference.rst

```diff
@@ -440,6 +440,27 @@ The WITH clause supports the following properties:
     :start-line: 2
     :end-line: 6
 
+.. _insert-into:
+
+INSERT INTO
+-----------
+
+**Synopsis**
+
+.. code:: sql
+
+    INSERT INTO stream_name
+      SELECT select_espr [., ...]
+      FROM from_item
+      [ WHERE condition ]
+      [ PARTITION BY column_name ];
+
+**Description**
+
+Stream the result of the SELECT query into an existing stream and its underlying topic.
+The schema and partitioning column produced by the query must match the stream's schema
+and key, respectively.
```

Comment:
What happens if it doesn't match? Does KSQL fail the query, or will you get undefined results? Worth clarifying the semantic here.